lib/linguist/vendor.yml

```diff
@@ -60,10 +60,17 @@
 # Bootstrap css and js
 - (^|/)bootstrap([^.]*)\.(js|css|less|scss|styl)$
 - (^|/)custom\.bootstrap([^\s]*)(js|css|less|scss|styl)$
+# Bootstrap with a directory between the files and the main folder like bootstrap/css/bootstrap.css
+- (^|/)bootstrap/(css|js)/.*\.(js|css|less|scss|styl)$
```

Comment:
This shouldn't be necessary as the regex on line 61 should catch `bootstrap.css` where ever it occurs in the path.

```ruby
irb(main):001:0> require 'yaml'
=> true
irb(main):002:0> vendored_paths = YAML.load_file(File.expand_path("../lib/linguist/vendor.yml", __FILE__))
=> # ... truncated for brevity ...
irb(main):003:0> VendoredRegexp = Regexp.new(vendored_paths.join('|'))
=> # ... truncated for brevity ...
irb(main):006:0> "/path/to/bootstrap/css/bootstrap.css" =~ VendoredRegexp
=> 8
irb(main):007:0>
```

Reply:
I'm not that good with Regex but since that regex didn't catch my bootstrap files I thought of the regex differently.